src/index.scss

```diff
@@ -1,5 +1,9 @@
 @import './assets/sass/theme.scss';
 
+body {
```

Comment:
Fixes issue where app text/images are not selecatable when click-drag && Ctrl+A to select regions